robotnik_msgs/State state
int32[] ids_recorded
int32[] ids_detected
